content/en/docs/reference/using-api/api-overview.md

```diff
@@ -18,95 +18,100 @@ This page provides an overview of the Kubernetes API.
-  (for example, `apiVersion: batch/v1`). You can find the full list of supported API groups in
-  [Kubernetes API reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/).
+*  The *core* (also called *legacy*) group is found at REST path `/api/v1`.
+   The core group is not specified as part of the `apiVersion` field, for
+   example, `apiVersion: v1`.
+*  The named groups are at REST path `/apis/$GROUP_NAME/$VERSION` and use
+   `apiVersion: $GROUP_NAME/$VERSION` (for example, `apiVersion: batch/v1`).
+   You can find the full list of supported API groups in
+   [Kubernetes API reference](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/).
 
-The two paths that support extending the API with [custom resources](/docs/concepts/extend-kubernetes/api-extension/custom-resources/) are:
+## Enabling or disabling API groups   {#enabling-or-disabling}
 
-- [CustomResourceDefinition](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
-  for basic CRUD needs.
-- [aggregator](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md) for a full set of Kubernetes API semantics to implement their own API server.
+Certain resources and API groups are enabled by default. You can enable or
+disable them by setting `--runtime-config` on the API server.  The
+`--runtime-config` flag accepts comma separated `<key>=<value>` pairs
+describing the runtime configuration of the API server. For example:
 
-## Enabling or disabling API groups
-
-Certain resources and API groups are enabled by default. You can enable or disable them by
-setting `--runtime-config` on the API server. The `--runtime-config` flag accepts comma separated
-key=value pairs describing the runtime configuration of the API server. For example:
-
-- to disable batch/v1, set `--runtime-config=batch/v1=false`
-- to enable batch/v2alpha1, set `--runtime-config=batch/v2alpha1`
+ - to disable `batch/v1`, set `--runtime-config=batch/v1=false`
+ - to enable `batch/v2alpha1`, set `--runtime-config=batch/v2alpha1`
 
 {{< note >}}
-When you enable or disable groups or resources, you need to restart the API server and controller-manager
-to pick up the `--runtime-config` changes.
+When you enable or disable groups or resources, you need to restart the API
+server and controller manager to pick up the `--runtime-config` changes.
 {{< /note >}}
+
+## Persistence
+
+Kubernetes stores its serialized state in terms of the API resources by writing them into
+{{< glossary_tooltip term_id="etcd" >}}.
+
+## {{% heading "whatsnext" %}}
+
+- Learn more about [API conventions](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#api-conventions)
+- Read the design documentation for
+  [aggregator](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md)
```

Comment:
Changes look good. Do you want to link to the aggregator in the what's next.
I think you removed that section from this page?

Reply:
We have the link to it design doc here. The two ways of extending k8s API have been moved over to the concepts section.